starter-code/app.js

```diff
@@ -8,6 +8,13 @@ const hbs          = require('hbs');
 const mongoose     = require('mongoose');
 const logger       = require('morgan');
 const path         = require('path');
+const session      = require('express-session');
+const Mongostore   = require('connect-mongo')(session)
+const flash        = require('connect-flash');
+const passport     = require('passport');
```

Comment:
So here you want to require('./auth/passport'), which is the file that has the Local Strategy defined